pkg/install/install_test.go

```diff
@@ -24,6 +30,22 @@ func testFillInTemplates(t *testing.T, expectedManifestCount int, params Templat
 			}
 		}
 	}
+
+	return manifests
+}
+
+func testExtractDeployment(t *testing.T, data []byte) *v1.Deployment {
+	manifest := string(data)
+	manifest = strings.Replace(manifest, "---\n", "", -1)
```

Comment:
Why is this needed?

Reply:
(I think it should work without it)

Reply:
Its not needed. I was having another issue at the time. Removed.